prereise/gather/hiflddata/data_trans.py

```diff
@@ -0,0 +1,645 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
```

Comment:
I believe these lines are unnecessary. @rouille, what do you think?

Reply:
I agree

Reply:
will remove